models/access.go

```diff
@@ -124,13 +125,20 @@ func (u *User) GetRepositoryAccesses() (map[*Repository]AccessMode, error) {
 // GetAccessibleRepositories finds repositories which the user has access but does not own.
 // If limit is smaller than 1 means returns all found results.
 func (user *User) GetAccessibleRepositories(limit int) (repos []*Repository, _ error) {
-	sess := x.Where("owner_id !=? ", user.ID).Desc("updated_unix")
+	sess := x.Where("owner_id !=? ", user.ID).Desc("uid").Desc("updated_unix")
```

Comment:
For a better readability and to get the real line / instruction in case of error, please set  XORM instruction per line

eg :

```
x.Where("owner_id !=? ", user.ID).
        Desc("uid").
        Desc("updated_unix")
```
or

```
x.
        Where("owner_id !=? ", user.ID).
        Desc("uid").
        Desc("updated_unix")
```